docs/k8s/deployment-guide.mdx

```diff
@@ -0,0 +1,165 @@
+
+See the [helm value `region`](https://github.com/ngrok/kubernetes-ingress-controller/blob/main/helm/ingress-controller/README.md#controller-parameters) to configure a specific region for the controller to use.
+
+## Metrics
+
+This operator exposes prometheus metrics on the `/metrics` endpoint. The metrics are exposed on the `:8080` port and can be scraped by prometheus or other services using typical means.
+
+This project is built using kube-builder, so out of the box it exposes the metrics listed [here](https://book.kubebuilder.io/reference/metrics-reference.html?highlight=metrics#default-exported-metrics-references)
+
+## White Label Agent Ingress
+
+If you're running the ngrok operator behind a corporate firewall, you may encounter connectivity issues due to the agent or controller not being able to communicate with the ngrok network edge. If you're facing such issues, you can run the ngrok cli command, which is documented [here](/guides/running-behind-firewalls) to diagnose the problem.
+
+However, if opening connectivity is not an option, you can set up a custom ingress domain on your dashboard. This way, you can configure the operator to use the custom domain instead of the default ngrok.app domain.
+
+To get started with this, go to your dashboard and create a custom ingress domain. Once created, you can configure the operator by using the following command:
+
+```bash
+helm install my-ingress-controller ngrok/kubernetes-ingress-controller \
+  --set serverAddr="ngrok.mydomain.com:443"
+```
+
+## Multiple Ingress Controller Installations
+
+::::note
+This capability only applies to the Ingress Controller
+::::
+
+The Ngrok Kubernetes Ingress Controller supports the Kubernetes concept of [Ingress Classes](https://kubernetes.io/docs/concepts/services-networking/ingress/) which allows you to run multiple ingress controllers in the same cluster. This feature is useful if you want to run multiple versions of the Ngrok Kubernetes Ingress Controller or other ingress controllers in the same cluster.
+
+### What is Ingress Class Filtering?
+
+Ingress controllers work by watching Kubernetes Ingress resources for changes and reconciling them to provide ingress to services. When multiple ingress controllers are installed in the same cluster, they can both try to reconcile all ingress objects by default, which can cause conflicts and unexpected behavior. To address this issue, Ingress Classes can be set on ingress objects and controllers can filter ingresses to only those that match their ingress class.
+
+By default, the Ngrok Kubernetes Ingress Controller creates a non-default ingress class with the name `ngrok`, which the controller watches for changes. In order for the controller to reconcile an ingress, the ingress must have the same ingress class as the controller.
+
+```yaml
+spec:
+  ...
+  ingressClassName: ngrok
+  rules:
+  ...
+```
```

Comment:
This is a rather technical controller gotcha that might require may require more context.

Reply:
This entire section is about what happens if you run multiple ingress controllers, so I think it provides context unless you think there is something specific that is missing?